### ISSUE TITLE
Switch from misaka to something else?

### ISSUE THREAD

**Issue by fluffy-critter** [isso-comments/isso#1040]
## Checklist
- [x] I am running the latest version. Installing Isso from GitHub from the `master` branch does not fix my issue
- [x] I have checked the [troubleshooting guide](https://isso-comments.de/docs/guides/troubleshooting/)
- [x] I have searched the [open issues](https://github.com/isso-comments/isso/issues), but my issue has not already been reported

## What is not working?

Misaka is no longer being maintained (nor is hoedown, its underlying C-based implementation) and also causes some installation headaches on some platforms. It would be beneficial to move to a pure-Python (and actively-maintained) Markdown processor.

I've been experimenting with [mistune](https://pypi.org/project/mistune/) for my newer projects and it presents a very similar API to misaka, so it shouldn't be particularly difficult for isso to switch to this.

**Comment by jelmer** [comment]
Yeah, mistune looks like the most reasonable alternative today. Are you volunteering to do the migration?

cc @ix5 in case they have any objections.

**Comment by fluffy-critter** [comment]
I'm not volunteering, just suggesting. I don't have a lot of energy for working on software these days.
